app/init.php

```diff
@@ -53,6 +53,7 @@
 use Utopia\Storage\Device;
 use Utopia\Storage\Storage;
 use Utopia\Storage\Device\Local;
+use Utopia\Storage\Device\BackBlaze;
```

Comment:
We should be consistent and spell `Backblaze` instead of `BackBlaze`

Reply:
agreed let me fix this in the utopia repo as well.

Reply:
done waiting for a merge there : https://github.com/utopia-php/storage/pull/46.

Reply:
addressed